interfaces/cython/cantera/composite.py

```diff
@@ -230,7 +230,10 @@ def __init__(self, phase, mass=None, moles=None, constant='UV'):
         else:
             self.mass = 1.0
 
-        assert constant in ('TP','TV','HP','SP','SV','UV')
+        if constant not in ('TP','TV','HP','SP','SV','UV'):
+            raise ValueError(
+                f"Constant {constant} is invalid."
```

Comment:
One more tiny change: 
```suggestion
                f"Constant {constant} is invalid. "
```